.pipeline/blubber.yaml

```diff
@@ -0,0 +1,29 @@
+version: v3
+base: docker-registry.wikimedia.org/nodejs-slim
+lives:
+  in: /srv/service
+runs:
+  environment: { APP_BASE_PATH: /srv/service }
+
+variants:
+  build:
+    base: docker-registry.wikimedia.org/nodejs-devel
+    apt: { packages: [git, python-pkgconfig, build-essential] }
+    node: { requirements: [package.json]}
+    runs: { environment: { LINK: g++ } }
+  development:
+    includes: [build]
+    entrypoint: [node, server.js]
+  test:
+    includes: [build]
+    apt: { packages: [libsqlite3-0] }
+    entrypoint: [npm, test]
+    runs: { environment: { TEST_TARGET: sqlite, TEST_MODE: multi_process } }
+  prep:
+    includes: [build]
+    node: { env: production }
+  production:
+    copies: prep
+    node: { env: production }
+    apt: { packages: [ca-certificates] }
```

Comment:
Does restbase reach out to any TLS enabled (and outside of wikimedia) endpoints ? If not, this isn't required.

Reply:
Outside prod no, but isn't it the plan to have all connections in prod TLS'ed as well?

Reply:
It is, but this package installs well-known x509 certs to be trusted by the system, not our own puppet CA (which is/will be used for this).

Reply:
Duh, good point! Will remove.